src/Pages/Profile.js

```diff
@@ -42,6 +42,7 @@ class Profile extends Component {
               : <ViewPersonalProfile
                   profile={this.state.profile}
                   handlePersonalClick={this.handlePersonalClick}
+                  refresh={this.props.refresh}
```

Comment:
I think this `refresh` function that you are passing around is kind of "anti-ReactJS".  We should not be doing anything to "manually" refresh our components.  When the component's state changes, it will re-render itself and it's child components that receive props.   

I'm curious why you are calling this here ?

@jscotty723 @l1ngoman @kristenmccloud @ChristopherJStubbs